app/models/starred.rb

```diff
@@ -0,0 +1,17 @@
+class Starred
+  attr_reader :name, :description, :language, :updated_at
+  
+  def initialize(params)
+    @name = params[:name]
+    @description = params[:description]
+    @language = params[:language]
+    @updated_at = params[:updated_at]
+  end
+
+  def self.all(current_user)
+    starred_repos_hashes = GithubService.starred(current_user)
+    starred_repos_hashes.map do |starred|
```

Comment:
Agreed. I'm gonna do a refactor later tonight that should get rid of these horrible names.